lte/gateway/c/core/oai/tasks/nas5g/src/ies/M5GUESecurityCapability.cpp

```diff
@@ -98,7 +99,11 @@ int UESecurityCapabilityMsg::DecodeUESecurityCapabilityMsg(
     ue_sec_capability->eia7 = *(buffer + decoded) & 0x1;
     decoded++;
 
-    // decoded = type_len + length_len + ue_sec_capability->length;
+    // Skipping the remaining bytes as not supported
```

Comment:
Statements at https://github.com/magma/magma/pull/12179/files#diff-3a0779f7713eb9082f1c0c9d4bde04e33585ac81524a9c38507d8f3ebaafe3eeR77 can be removed

Reply:
Addressed Review Comment